### PR TITLE
Adds absolute paths for maxEntScan files and brca.zcore.json file

### PR DESCRIPTION
**Pull request by ccdlott**
Adds back tests skipped that depend on score3.pl and score5.pl

### CODE REVIEW

File: pipeline/splicing/brca.zscore.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"donors": {"std": 2.3289956850167082, "mean": 7.9380909090909073}, "acceptors": {"std": 2.4336623152078452, "mean": 7.984909090909091}}
```
Comment (zfisch) [inline pipeline/splicing/brca.zscore.json]: needs newline at bottom

**Comment by zfisch** [comment]
@ccdlott still getting local failures on these tests. Let's get your local machine set up and make sure these all pass before merging (it's less of a concern after we've deployed to beta later this week).

**Comment by ccdlott** [comment]
Okay, do you have some time to work on this tomorrow or do you want to wait until after everything is deployed to beta?

**Comment by zfisch** [comment]
I can definitely help with this tomorrow -- let's coordinate on slack

**Pull request closed without merging**